test/replcompletions.jl

```diff
@@ -494,6 +493,29 @@ c, r, res = test_scomplete(s)
         ENV["PATH"] = oldpath
     end
 
+    # Make sure completion results are unique in case things are in the env path twice.
+    let
+        # Create /tmp/repl-completion and /tmp/repl-completion-subdir/repl-completion
+        file0 = joinpath(tempdir(), "repl-completion")
+        dir = joinpath(tempdir(), "repl-completion-subdir")
+        mkdir(dir)
+        file1 = joinpath(dir, "repl-completion")
+        touch(file0)
+        touch(file1)
+
+        oldpath = ENV["PATH"]
+        ENV["PATH"] = string(tempdir(), ":", dir)
```

Comment:
I think I'll go with just using the nightly binary (once this is part of them). I might also even get around rebuilding the sysimg, as I only plan to change the tests.